lib/build.gradle.kts

```diff
@@ -1,42 +1,27 @@
+import buildsrc.utils.Rife2TestListener
 import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 import org.gradle.api.tasks.testing.logging.TestLogEvent
 import org.jetbrains.dokka.gradle.DokkaTask
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
-import java.net.URI
-import java.net.http.HttpClient
-import java.net.http.HttpRequest
-import java.net.http.HttpResponse
 
 
 plugins {
     id("application")
-    id("com.github.ben-manes.versions") version "0.44.0"
-    id("io.gitlab.arturbosch.detekt") version "1.22.0"
+    id("com.github.ben-manes.versions")
+    id("io.gitlab.arturbosch.detekt")
     id("java-library")
     id("maven-publish")
-    id("org.jetbrains.dokka") version "1.7.20"
-    id("org.jetbrains.kotlin.jvm") version "1.8.0"
-    id("org.jetbrains.kotlinx.kover") version "0.6.1"
-    id("org.sonarqube") version "3.5.0.2730"
-    id("signing")
+    id("org.jetbrains.kotlin.jvm")
```

Comment:
version & group are unspecified

todo: create a `kotlin-jvm` convention plugin that applies the `builsrc-base` convention